cluster/addons/cluster-monitoring/influxdb/influxdb-grafana-controller.yaml

```diff
@@ -41,10 +41,8 @@ spec:
               cpu: 100m
               memory: 100Mi
           env: 
-            - name: INFLUXDB_EXTERNAL_URL
-              value: /api/v1/proxy/namespaces/kube-system/services/monitoring-influxdb:api/db/
```

Comment:
@vishh before this PR, grafana is going through kube-proxy to talk to influxdb (which is running on the same Pod) as per RC. With this change it will just use localhost to talk to influxdb. It avoids having to go out to kube-proxy. The "/api/v1/proxy" URL format is (AFAIK) a proxy that runs in the kube-apiserver and issues a redirect.